styles.css

```diff
@@ -0,0 +1,595 @@
+}
+
+.social-media-icons .far {
+  color: rgb(102, 102, 102);
+  font-size: 22px;
+  font-weight: 50;
+  background-color: transparent;
+  border: none;
+}
+
+.social-media-icons .far:hover {
+  color: #eee;
+  cursor: pointer;
+}
+
+.main-text {
+  font-size: 20px;
+  line-height: 30px;
+  color: #333;
+  margin-bottom: 20px;
+}
+
+.separation {
+  width: 72px;
+  border-bottom: 1px solid rgb(204, 204, 204);
+  margin: 0 auto 18px auto;
+}
+
+ul {
+  list-style: none;
+  font-size: 15px;
+  line-height: 19px;
+  text-align: center;
+  font-weight: bold;
+}
+
+.subscribe {
+  display: block;
+  width: 169px;
+  margin: 3px auto 18px auto;
+  color: #326891;
+  font-size: 15px;
+  line-height: 19px;
+  text-decoration: none;
+  font-weight: bold;
+}
+
+/* -------------------- Aside Section --------------- */
+
+aside {
+  width: 210px;
+  margin-left: 30px;
+}
+
+.aside-articles {
+  width: 100%;
+  position: relative;
+  top: 1000px;
+  z-index: 10;
+}
+
+.aside-title {
+  border-bottom: 1px solid #eee;
+  padding-bottom: 10px;
+  margin-bottom: 10px;
+}
+
+.picks article {
+  width: 100%;
+  height: 75px;
+  margin-bottom: 15px;
+}
+
+.right-div h3 {
+  padding: 12px 0;
+  font-weight: 400;
+  border-top: 1px solid lightgrey;
+}
+
+.picks a h3 {
+  font-size: 14px;
+  font-weight: lighter;
+  max-width: 125px;
+  margin-left: 10px;
+  color: #000;
+}
+
+.img-div {
+  min-width: 75px;
+  min-height: 75px;
+}
+
+.img-div img {
+  width: 75px;
+  height: 75px;
+}
+
+/* -------------------- THE SECOND MAIN ----------------------- */
+
+.second-main {
+  width: 100%;
+  margin: 0 auto;
+}
+
+.container-article {
+  max-width: 1000px;
+  width: 100%;
+  margin: 0 auto;
+}
+
+.second-main h1 {
+  font-size: 20px;
+  font-family: georgia, "times new roman", times, serif;
+}
+
+.wrapper {
+  display: grid;
+  width: 1000px;
+  grid-template-columns: repeat(4, 1fr);
+  grid-template-rows: repeat(4, 155px);
+}
+
+.description p {
+  margin-right: 20px;
+}
+
+.box1 {
+  grid-column-start: 1;
+  grid-column-end: 5;
+  grid-row-start: 1;
+  grid-row-end: 2;
+}
+
+.box2 {
+  grid-column-start: 1;
+  grid-column-end: 5;
+  grid-row-start: 2;
+  grid-row-end: 3;
+  display: grid;
+  width: 1000px;
+  grid-template-columns: repeat(4, 1fr);
+}
+
+.box3 {
+  grid-column-start: 1;
+  grid-column-end: 5;
+  grid-row-start: 3;
+  grid-row-end: 4;
+}
+
+.box4 {
+  grid-column-start: 1;
+  grid-column-end: 5;
+  grid-row-start: 4;
+  grid-row-end: 5;
+  display: grid;
+  width: 1000px;
+  grid-template-columns: repeat(4, 1fr);
+}
+
+.second-footer {
+  margin-top: 50px;
+}
+
+.second-footer::before {
+  width: 60px;
+  content: "";
+  display: inline-block;
+  height: 1px;
+  margin: 0 0 13px;
+  background-color: #000;
+}
+
+.second-footer p {
+  margin-bottom: 100px;
+  font-size: 13px;
+}
+
+.video-container {
+  width: 100%;
+  height: 0;
+  padding-bottom: 56.25%;
+  position: relative;
+  z-index: 10;
+}
+
+.video-container iframe {
+  width: 100%;
```

Comment:
- [ ] Remove all the unnecessarily repeated CSS code. Keeping your code DRY is an excellent practice that will help you a lot and make your code more professional. (Check this link form better understanding [Good and Bad CSS Practices for Beginners](https://speckyboy.com/good-bad-css-practices/)) ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/html_css/section/css_bad_practices_requirement.png) from Pathwright)